ruby/red-arrow/lib/arrow/array.rb

```diff
@@ -73,5 +73,18 @@ def reverse_each
     def to_arrow
       self
     end
+
+    alias_method :value_data_type_raw, :value_data_type
+    def value_data_type
+      @value_data_type ||= value_data_type_raw
+    end
+
+    # def to_a
+    #   values
+    # end
+
+    # def values
+    #   raw_values
+    # end
```

Comment:
Do you leave these intentionally?

Reply:
Oh, sorry.
This was included accidentally. I've removed this.
This is a part of the next change.